docs/content/preview/explore/query-1-performance/explain-analyze.md

```diff
@@ -106,37 +120,45 @@ yugabyte=# EXPLAIN ANALYZE SELECT * FROM employees a LEFT JOIN LATERAL
-   Join Filter: (a.* = b.*)
-   Rows Removed by Join Filter: 6
-   ->  Seq Scan on employees_k a  (cost=0.00..100.00 rows=1000 width=112) (actual time=1.747..1.749 rows=3 loops=1)
-   ->  Materialize  (cost=0.00..105.00 rows=1000 width=112) (actual time=0.155..0.157 rows=3 loops=3)
-         ->  Seq Scan on employees_k b  (cost=0.00..100.00 rows=1000 width=112) (actual time=0.450..0.454 rows=3 loops=1)
- Planning Time: 0.072 ms
- Execution Time: 2.938 ms
-(8 rows)
+                                                       QUERY PLAN
+------------------------------------------------------------------------------------------------------------------------
+ Merge Left Join  (cost=299.66..379.66 rows=5000 width=88) (actual time=2.382..2.389 rows=3 loops=1)
+   Merge Cond: (a.* = b.*)
+   ->  Sort  (cost=149.83..152.33 rows=1000 width=112) (actual time=1.485..1.500 rows=3 loops=1)
+         Sort Key: a.*
+         Sort Method: quicksort  Memory: 25kB
+         ->  Seq Scan on employees a  (cost=0.00..100.00 rows=1000 width=112) (actual time=0.407..0.410 rows=3 loops=1)
+   ->  Sort  (cost=149.83..152.33 rows=1000 width=112) (actual time=0.657..0.658 rows=3 loops=1)
+         Sort Key: b.*
+         Sort Method: quicksort  Memory: 25kB
+         ->  Seq Scan on employees b  (cost=0.00..100.00 rows=1000 width=112) (actual time=0.278..0.280 rows=3 loops=1)
+ Planning Time: 30.850 ms
+ Execution Time: 3.943 ms
+ Peak Memory Usage: 137 kB
+(13 rows)
 ```
 
 The server output from the preceding example includes the number of rescans (loops) each node performed, the number of milliseconds passed before the first row was returned, total time before the last tuple was returned by each execution node, and the number of tuples returned by each execution node.
 
 `EXPLAIN`, on the other hand,  does not provide this additional information, as shown in the following examples:
 
 ```sql
-yugabyte=# EXPLAIN SELECT * FROM employees_k a LEFT JOIN LATERAL
-(SELECT * FROM employees_k b WHERE a = b) c ON TRUE;
+yugabyte=# EXPLAIN SELECT * FROM employees a LEFT JOIN LATERAL
+(SELECT * FROM employees b WHERE a = b) c ON TRUE;
 ```
 
 ```output
-QUERY PLAN
-----------------------------------------------------------------------
-Nested Loop Left Join  (cost=0.00..15202.50 rows=5000 width=88)
-   Join Filter: (a.* = b.*)
-   ->  Seq Scan on employees_k a  (cost=0.00..100.00 rows=1000 width=112)
-   ->  Materialize  (cost=0.00..105.00 rows=1000 width=112)
-         ->  Seq Scan on employees_k b  (cost=0.00..100.00 rows=1000 width=112)
-(5 rows)
+                                  QUERY PLAN
+------------------------------------------------------------------------------
+ Merge Left Join  (cost=299.66..379.66 rows=5000 width=88)
+   Merge Cond: (a.* = b.*)
+   ->  Sort  (cost=149.83..152.33 rows=1000 width=112)
+         Sort Key: a.*
+         ->  Seq Scan on employees a  (cost=0.00..100.00 rows=1000 width=112)
+   ->  Sort  (cost=149.83..152.33 rows=1000 width=112)
```

Comment:
Significantly higher-cost example. Is this on purpose?

Reply:
This example shows the differences between an _EXPLAIN_ output versus an _EXPLAIN ANALYZE_ output where ANALYZE provides additional details.